### PR TITLE
Onboard LAADS DAAC Geoloco Service

### PR DESCRIPTION
**Pull request by cbrownstevenson**
## Jira Issue ID
HARMONY-1523

## Description
Onboarding LAADS DAAC Geoloco Service to Harmony

## Local Test Steps
Locally tested with the user Jupyter notebooks in https://git.earthdata.nasa.gov/projects/LDDS/repos/laads-geoloco-service/browse/docs

## PR Acceptance Checklist
* [ x] Acceptance criteria met
* [ ] Tests added/updated (if needed) and passing
* [ x] Documentation updated (if needed)

### CODE REVIEW

**Review by chris-durbin — APPROVED** [review]
I tested this out successfully in a deployed sandbox environment.

File: config/services.yml [hunk 1]
```diff
@@ -349,6 +349,35 @@ https://cmr.earthdata.nasa.gov:
...
+          STAGING_PATH: public/ldds/geoloco
+    umm_s: S1259233619-LAADSCDUAT
+    capabilities:
+       #concatenation:true
+       #concatenate_by_default: true
+       subsetting:
+         bbox: true
+         variable: true
+         multiple_variable: true
+         shape: true
```
[the remark this answers is not in the document]
Comment (cbrownstevenson) [reply config/services.yml:375]: It does, but it's not utilized on premises so I can remove it.

File: config/services.yml [hunk 2]
```diff
@@ -349,6 +349,35 @@ https://cmr.earthdata.nasa.gov:
...
+          and geographic projection).
+    data_operation_version: '0.18.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/ldds/geoloco
+    umm_s: S1259233619-LAADSCDUAT
```
[the remark this answers is not in the document]
Comment (indiejames) [reply config/services.yml:366]: How is it returning multiple output files?

**Review by indiejames — APPROVED** [review]

File: config/services.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (cbrownstevenson) [reply config/services.yml:366]: It separates the granule and/or variables by variables/bands/planes if reprojecting/regridding. So one slice of data per file if the dataset is above 2 dimensions.

**Pull request merged**